.github/workflows/create-release.yml

```diff
@@ -0,0 +1,40 @@
+name: Create GH release
+
+# Create a GitHub release for each new tag.
+# The release notes are taken from the release notes file
+# modified in that commit located in Documentation/RelNotes directory.
+
+on:
+  push:
+    tags:
+      - v[0-9]+.*
+
+permissions:
+  contents: write
+
+jobs:
+  create-gh-release:
+    name: Create a new release or update an existing release in the GitHub repository
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2  # OR '0' To retrieve all preceding commit.
```

Comment:
Why 2? And why ddo we need that comment?

Reply:
The comment is not needed. The action only fetches a single revision by default (`fetch-depth: 1`). To get the list of modified files, *tj-actions/changed-files@v11.7* needed last two revisions.

Reply:
That will give you the list of files modified in the tip commit. Not between tags.

Reply:
Yes, exactly.